actions/checks.security.groups.yaml

```diff
@@ -0,0 +1,40 @@
+description: Check Security groups in projects for specific parameters
+enabled: true
+entry_point: src/openstack_check_actions.py
+name: checks.security.groups
+parameters:
+  submodule:
+    default: security_groups_check
+    immutable: true
+    type: string
+  cloud_account:
+    description: The clouds.yaml account to use whilst performing this action
+    default: ""
+    required: true
+    type: string
+  ip_prefix:
+    description: The IPs that are allowed to access the port range specified below
```

Comment:
Are these IPs comma separated?

Reply:
No, these are given as a CIDR e.g. 0.0.0.0/0 for all internet.
I'll make this clearer.